planetcantile/data/generate.py

```diff
@@ -41,42 +44,43 @@ def __post_init__(self):
         # try to get the geographic_crs
         geographic_crs = self.crs.geodetic_crs
 
+# Grab the wkts that mirror the OCG source and parse out just the GeogCRS in a hacktastic way.
+with urllib.request.urlopen('https://raw.githubusercontent.com/pdssp/planet_crs_registry/main/data/result.wkts') as response:
+    resp = response.read().decode(response.headers.get_content_charset())
 
-gcs_mars_2000_ellipsoid = 'ELLIPSOID["Mars_2000_Sphere_IAU_IAG",3396190,0.0,LENGTHUNIT["metre",1]],ID["ESRI",106971]]'
-gcs_mars_2000_sphere_datum = f'DATUM["D_Mars_2000_Sphere",{gcs_mars_2000_ellipsoid},PRIMEM["Reference_Meridian",0,ANGLEUNIT["degree",0.0174532925199433,ID["EPSG",9122]]]'
-gcs_mars_2000_sphere = CRS(f"""GEOGCRS["GCS_Mars_2000_Sphere",{gcs_mars_2000_sphere_datum},CS[ellipsoidal,2],AXIS["latitude",north,ORDER[1],ANGLEUNIT["degree",0.0174532925199433,ID["EPSG",9122]]],AXIS["longitude",east,ORDER[2],ANGLEUNIT["degree",0.0174532925199433,ID["EPSG",9122]]]]""")
+# Parse all the GeoCRSs from the list
+geocrss = []
+for wkt_str in resp.split(os.linesep + os.linesep):
+    if 'GEOGCRS' in wkt_str[:7] and 'TRIAXIAL' not in wkt_str:  # insanely hacky
+        geocrss.append(wkt_str)
 
-moon_2000_ellipsoid = 'SPHEROID["Moon_2000_IAU_IAG",1737400.0,0.0,LENGTHUNIT["metre",1]]'
-moon_2000_datum = f'DATUM["D_Moon_2000_Sphere",{moon_2000_ellipsoid},PRIMEM["Greenwich",0],UNIT["Decimal_Degree",0.0174532925199433]]'
-moon_2000_sphere = CRS(f'GEOGCS["Moon_2000",{moon_2000_datum},CS[ellipsoidal,2],AXIS["latitude",north,ORDER[1],ANGLEUNIT["degree",0.0174532925199433,ID["EPSG",9122]]],AXIS["longitude",east,ORDER[2],ANGLEUNIT["degree",0.0174532925199433,ID["EPSG",9122]]]]')
+# Build a dynamic crss list
+crss = []
+for crs in geocrss:
+    crs_obj = CRS(crs)
+    title = crs_obj.name
+    
+    # There has to be an easier way to get at the id?
```

Comment:
I think calling `to_authority()` could be what you want here. For `"Sun (2015) - Sphere / Ocentric/ Equirectangular, clon = 0"` I get a tuple of `('IAU_2015', '1010')` which matches the content of 'id' `{'authority': 'IAU', 'code': 1010, 'version': 2015}` pretty well, would just need to split the first string. There is also a AuthorityMatchInfo class returned by the `list_authority()` method